notebooks/env.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<p>Please go through and make sure the language is good, maybe put it through Grammarly or ChatGPT. eg</p><blockquote>The <code>terminated</code> is <code>True</code> if environment terminates, like task completion. </blockquote><p>is a bit awkwardly written.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chandar-lab/RLHive/pull/305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>